src/workflow/reset.c

```diff
@@ -14,7 +14,6 @@
 
 #include "reset.h"
 #include "confirm.h"
-#include "password.h"
 #include "workflow.h"
 
 #include <reset.h>
```

Comment:
Since we're here, `reset.h` seems to be included twice, or is this a different reset from the `"reset.h"` above?

Reply:
yeah, src/reset.h contains `reset_reset()`.

Reply:
done

Reply:
ah, one if workflow_reset and the other one is reset_reset. this is where include with the path would've made it more clear:

```
#include "reset.h"
#include "workflow/reset.h"
```

(both relative to src/)

but ok, not relevant to this PR.